server/server.js

```diff
@@ -3,15 +3,15 @@ const appConfig = require('./app');
 const channels = require('./channels/index');
 const logger = require('./logging');
 const Raven = require('raven');
-const mongodbConfig = require('./models/essentials');
+const { applyMiddlewares } = require('./channels/index');
```

Comment:
`./channels/index` is imported twice.

Either do `channels.applyMiddlewares` or `{ applyMiddlewares, listen }`.